src/rpc/blockchain.cpp

```diff
@@ -390,6 +397,8 @@ UniValue getblock(const UniValue& params, bool fHelp)
             "  \"bits\" : \"1d00ffff\", (string) The bits\n"
             "  \"difficulty\" : x.xxx,  (numeric) The difficulty\n"
             "  \"chainwork\" : \"xxxx\",  (string) Expected number of hashes required to produce the chain up to this block (in hex)\n"
+            "  \"sizelimit\" : n,       (numeric) BIP100 block size limit as of this block\n"
+            "  \"sizelimitvote\" : n,   (numeric) This block's sizelimit vote\n"
```

Comment:
Wording is corrected in next push.  Other than getblocktemplate, whose sizelimit has always referred to the prospective next block, all of the RPC sizelimit and sizelimitvote attributes refer to the block itself.  This is for consistency with the other attributes, especially difficulty, which is very analogous to sizelimit under bip100.

An additional attribute could be added for next-block sizelimit, but this would help the other implementations much more than BU.  In BU with -bip100 set, the next-block sizelimit is always available as EB.